packages/shared/components/popups/LedgerTransaction.svelte

```diff
@@ -0,0 +1,86 @@
+<script lang="ts">
+    import { Illustration, Text } from 'shared/components'
+    import { displayNotifications, showAppNotification } from 'shared/lib/notifications';
+    import { formatUnitBestMatch } from 'shared/lib/units';
+    import { get } from "svelte/store";
```

Comment:
Perhaps we should use single quotes for consistency?

`import { get } from "svelte/store";` -> `import { get } from 'svelte/store';`

Reply:
Also we have not been using `;` to end lines

Reply:
Ah yes, I've just fixed this. I need to double check my Webstorm configuration for JS / TS.

Reply:
Adding a linter check in a Git hook or a CI pipeline might be nice to enforce some formatting consistency